tests/commands/delete/test_delete_cmd.py

```diff
@@ -104,13 +104,28 @@ def test_delete_user(empty_mock_app, user_obj):
     store.user_collection.insert_one(user_obj)
     assert store.user_collection.find_one()
 
+    ## And the user is an assignee of a case of an institute
+    store.institute_collection.insert_one(institute_obj)
+
+    ## And no events in the database
+    assert store.event_collection.find_one() is None
+
+    case_obj["assignees"] = [user_obj["email"]]
+    store.case_collection.insert_one(case_obj)
+    assert store.case_collection.find_one({"assignees": {"$in": case_obj["assignees"]}})
+
     ## WHEN deleting the user from the CLI
     result = runner.invoke(cli, ["delete", "user", "-m", user_obj["email"]])
 
     ## THEN the user should be gone
     assert result.exit_code == 0
     assert store.user_collection.find_one() is None
 
+    ## The case should not have an assignee
+    assert store.case_collection.find_one({"assignees": {"$in": case_obj["assignees"]}}) is None
+    ## And a new event for unassigning the user should have been created in event collection
+    assert store.event_collection.find_one()
+
```

Comment:
Ran into an error when running Pytest. Maybe I missed something in the setup?

```
$ pytest commands/delete/
==================================================================== test session starts =====================================================================
platform darwin -- Python 3.6.8, pytest-5.4.1, py-1.8.0, pluggy-0.13.1 -- /miniconda3/envs/py3.6/bin/python
cachedir: .pytest_cache
rootdir: /Users/Mikael/Work/codereview/scout, inifile: pytest.ini
plugins: mock-3.1.0, flask-0.11.0, cov-2.7.1
collected 15 items

commands/delete/test_delete_cmd.py::test_delete_panel_non_existing PASSED                                                                              [  6%]
commands/delete/test_delete_cmd.py::test_delete_panel PASSED                                                                                           [ 13%]
commands/delete/test_delete_cmd.py::test_delete_index PASSED                                                                                           [ 20%]
commands/delete/test_delete_cmd.py::test_delete_nonexisting_user PASSED                                                                                [ 26%]
commands/delete/test_delete_cmd.py::test_delete_user ERROR                                                                                             [ 33%]

=========================================================================== ERRORS ===========================================================================
_____________________________________________________________ ERROR at setup of test_delete_user _____________________________________________________________

request = <SubRequest 'parsed_case' for <Function test_delete_user>>
scout_config = {'analysis_date': datetime.datetime(2016, 10, 12, 14, 0, 46), 'chromograph_image_files': 'scout/demo/images', 'chromograph_prefixes': {'chr': 'cytoband.txt.chr', 'roh': 'roh_chr', 'upd': 'upd_chr'}, 'default_gene_panels': ['panel1'], ...}

    @pytest.fixture(scope="function")
    def parsed_case(request, scout_config):
        """Get the lines for a case"""
>       case = parse_case(scout_config)

conftest.py:318:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/miniconda3/envs/py3.6/lib/python3.6/site-packages/scout_browser-4.18-py3.6.egg/scout/parse/case.py:508: in parse_case
    add_smn_info_case(case_data)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

case_data = {'analysis_date': datetime.datetime(2016, 10, 12, 14, 0, 46), 'assignee': None, 'case_id': 'internal_id', 'chromograph_image_files': 'scout/demo/images', ...}

    def add_smn_info_case(case_data):
        """Add SMN CN / SMA prediction from TSV files to individuals in a yaml load case context

        Args:
            case_data(dict)
        """

        if not case_data.get("smn_tsv"):
            LOG.warning("No smn_tsv though add_smn_info_case called. This is odd.")
            return

>       file_handle = open(case_data["smn_tsv"], "r")
E       FileNotFoundError: [Errno 2] No such file or directory: 'scout/demo/643594.solo.smn.tsv'

/miniconda3/envs/py3.6/lib/python3.6/site-packages/scout_browser-4.18-py3.6.egg/scout/parse/case.py:198: FileNotFoundError
------------------------------------------------------------------- Captured stdout setup --------------------------------------------------------------------




------------------------------------------------------------------- Captured stderr setup --------------------------------------------------------------------
2020-07-06 10:52:26 mikaellmbp conftest[30740] INFO Get a real pymongo client
2020-07-06 10:52:26 mikaellmbp conftest[30740] INFO Connecting to database realtestdb
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.index[30740] INFO creating indexes for hgnc_gene collection: build_chromosome, build_hgncid, build_aliases
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.index[30740] INFO creating indexes for variant collection: caseid_rankscore, caseid_variantrank, caseid_category_varianttype_rankscore, hgncsymbol_rankscore_category_varianttype, caseid_variantid, caseid_category_chromosome_start_end, sanger
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.index[30740] INFO creating indexes for hpo_term collection: number
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.index[30740] INFO creating indexes for event collection: category_verb, variant_id
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.index[30740] INFO creating indexes for transcript collection: hgncid_length
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.index[30740] INFO creating indexes for exon collection: build_hgncid
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.index[30740] INFO creating indexes for case collection: synopsis_text
2020-07-06 10:52:26 mikaellmbp conftest[30740] INFO Connected to database
2020-07-06 10:52:26 mikaellmbp scout.adapter.client[30740] INFO Try to connect to mongodb://localhost:27017
2020-07-06 10:52:26 mikaellmbp scout.adapter.client[30740] INFO Connection established
2020-07-06 10:52:26 mikaellmbp scout.adapter.mongo.base[30740] INFO connecting to database: localhost:27017/realtestdb
2020-07-06 10:52:26 mikaellmbp conftest[30740] INFO Building a institute
2020-07-06 10:52:26 mikaellmbp scout.build.institute[30740] INFO Building institute cust000 with display name test_institute
2020-07-06 10:52:26 mikaellmbp scout.parse.case[30740] DEBUG Checking for SMN TSV..
2020-07-06 10:52:26 mikaellmbp scout.parse.case[30740] INFO Adding SMN info from scout/demo/643594.solo.smn.tsv.
--------------------------------------------------------------------- Captured log setup ---------------------------------------------------------------------
INFO     conftest:conftest.py:545 Get a real pymongo client
INFO     conftest:conftest.py:566 Connecting to database realtestdb
INFO     scout.adapter.mongo.index:index.py:51 creating indexes for hgnc_gene collection: build_chromosome, build_hgncid, build_aliases
INFO     scout.adapter.mongo.index:index.py:51 creating indexes for variant collection: caseid_rankscore, caseid_variantrank, caseid_category_varianttype_rankscore, hgncsymbol_rankscore_category_varianttype, caseid_variantid, caseid_category_chromosome_start_end, sanger
INFO     scout.adapter.mongo.index:index.py:51 creating indexes for hpo_term collection: number
INFO     scout.adapter.mongo.index:index.py:51 creating indexes for event collection: category_verb, variant_id
INFO     scout.adapter.mongo.index:index.py:51 creating indexes for transcript collection: hgncid_length
INFO     scout.adapter.mongo.index:index.py:51 creating indexes for exon collection: build_hgncid
INFO     scout.adapter.mongo.index:index.py:51 creating indexes for case collection: synopsis_text
INFO     conftest:conftest.py:573 Connected to database
INFO     scout.adapter.client:client.py:56 Try to connect to mongodb://localhost:27017
INFO     scout.adapter.client:client.py:63 Connection established
INFO     scout.adapter.mongo.base:base.py:85 connecting to database: localhost:27017/realtestdb
INFO     conftest:conftest.py:461 Building a institute
INFO     scout.build.institute:institute.py:32 Building institute cust000 with display name test_institute
DEBUG    scout.parse.case:case.py:505 Checking for SMN TSV..
INFO     scout.parse.case:case.py:507 Adding SMN info from scout/demo/643594.solo.smn.tsv.
================================================================== short test summary info ===================================================================
ERROR commands/delete/test_delete_cmd.py::test_delete_user - FileNotFoundError: [Errno 2] No such file or directory: 'scout/demo/643594.solo.smn.tsv'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================= 4 passed, 1 error in 4.87s =================================================================
```

Reply:
Could it be that in your local branch the file scout/demo/643594.solo.smn.tsv went missing somehow?

Reply:
Could it be that in your local branch the file scout/demo/643594.solo.smn.tsv went missing somehow?

Reply:
Must be something on your local branch, because it's working here on github.. 🤔

Reply:
True that.

Reply:
Thanks for the review!